stl/src/pplerror.cpp

```diff
@@ -11,7 +11,7 @@
 #include <roerrorapi.h>
 #include <trnsctrl.h>
 
-extern "C" void* __GetPlatformExceptionInfo(int*);
+extern "C" void* __GetPlatformExceptionInfo(int*) noexcept;

```

Comment:
This is another `src/vctools/crt/vcruntime/src/eh/ehhelpers.cpp` function - should we be changing our declaration?

Reply:
Ditto "yes, but not in this PR"